ci/pipeline.yaml

```diff
@@ -260,6 +260,48 @@ jobs:
         status: failure
         context: check-types
 
+  - name: pr-test
```

Comment:
Maybe check with Yorick if we should do this with a github action instead

Reply:
yes I was about to when I am finished with preparations,thx